src/main/java/com/karl/http/HTTPStatus.java

```diff
@@ -0,0 +1,20 @@
+package com.karl.http;
+
+public enum HTTPStatus {
+  Ok("200", "OK"),
+  NotFound("404", "Not Found"),
+  MethodNotAllowed("405", "Not Found");
```

Comment:
`Not Found` --> `Method Not Allowed` for the `405`? I saw your comment about 404/405s not really being implemented according to the spec so maybe this is intentional due to this being sort of a WIP/refactor PR.